reana_client/cli/analyses.py

```diff
@@ -19,27 +19,40 @@
 # In applying this license, CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
-"""REANA client analysis related commands."""
+"""REANA client workflow related commands."""
 
 import logging
 
 import click
 
 from ..config import (default_organisation, default_user,
                       reana_yaml_default_file_path)
+
 from ..utils import load_reana_spec, load_workflow_spec
 
 
+@click.group(
+    help='All interaction related analysis on REANA cloud.')
```

Comment:
`All interaction related to analyses on REANA cloud.`

Reply:
Fixed